org.dash.dash-core.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.dash.dash-core</id>
+  <name>Dash Core</name>
+  <project_license>MIT</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>
+    Fully validating Dash peer-to-peer network node, wallet and GUI
+  </summary>
+  <url type="homepage">
+    https://dash.org/
+  </url>
+  <url type="bugtracker">
+    https://github.com/dashpay/dash/issues/new?title=[flatpak]
+  </url>
+  <launchable type="desktop-id">
+    org.dash.dash-core.desktop
+  </launchable>
+  <description>
+    <p>
+    Dash Core connects to the Dash peer-to-peer network to download and
+    fully validate blocks and transactions. It also includes a wallet and
+    graphical user interface.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">
+        https://docs.dash.org/en/stable/_images/overview.png
+      </image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release date="2024-10-22" version="21.1.1"/>
+  </releases>
+  <content_rating type="oars-1.1"/>
```

Comment:
is this rating correct? There should be something in the OARS generator related to money.

Reply:
Just went through the generator and appears correct. Only stuff financial is as shown below

![image](https://github.com/user-attachments/assets/f4ca3ca8-3662-49f2-87b4-e8fc1d162e43)

Reply:
Gambling should be "intense".

Reply:
Thanks for your response, can you provide any justification or documentation to support this? The application does not include any gambling functionality, and org.bitcoincore.bitcoin-qt is also marked as oars-1.1, the functionality in the two applications is very similar.

Reply:
Going by the screenshot, does the send/receive allow to send or receive cryptocurrencies? Then it should have `<content_attribute id="money-purchasing">intense</content_attribute>`

Reply:
Is there any documentation to support this? based on my view looking at:
- monero: https://github.com/flathub/org.getmonero.Monero/blob/master/extra/org.getmonero.Monero.metainfo.xml#L6C1-L7C1
- bitcoin: https://github.com/flathub/org.bitcoincore.bitcoin-qt/blob/master/org.bitcoincore.bitcoin-qt.metainfo.xml#L53
- exodus: https://github.com/flathub/io.exodus.Exodus/blob/master/io.exodus.Exodus.appdata.xml#L154

All three support sending / receiving cryptocurrency, and all three share the same OARS as this application currently. I cannot find any examples that show that this should qualify under "money-purchasing" but I also can't seem to find a definition anywhere besides in the generator itself? I'm not so opposed to changing it, but I'd prefer to see a definition for what exactly "money-purchasing" means

Reply:
I'd say all 3 should have it but they were submitted a long time ago when I was not here. Money purchasing is more like dealing with any transactions that can involve actual money or things that can be converted to money. 

Unfortunately we don't control the OARS standard and it's quite an old specification which isn't actively developed anymore. So the docs are missing and the categories do not reflect the real world usage today.

Reply:
https://github.com/flathub/flathub/pull/5747/commits/2dd1609f5206d38173bb47b652129cd76b408324